mava/configs/default/rec_mappo.yaml

```
@@ -3,7 +3,7 @@ defaults:
   - arch: anakin
   - system: ppo/rec_mappo
   - network: rnn  # [rnn, rcnn]
-  - env: rware  # [cleaner, connector, gigastep, lbf, mabrax, matrax, rware, smax]
+  - env: lbf  # [cleaner, connector, gigastep, lbf, mabrax, matrax, rware, smax, mpe]
```

lets leave the default as rware

```suggestion
  - env: rware  # [cleaner, connector, gigastep, lbf, mabrax, matrax, rware, smax, mpe]
```